property.go

```diff
@@ -0,0 +1,91 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string `json:"name"`
+	Schema string `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+	Name    string   `graphql:"name"`
+	Schema  string   `graphql:"schema"`
+}
+
+type PropertyDefinitionConnection struct {
+	Nodes      []PropertyDefinition
+	PageInfo   PageInfo
+	TotalCount int
+}
+
+func (client *Client) CreatePropertyDefinition(input PropertyDefinitionInput) (*PropertyDefinition, error) {
+	var m struct {
+		Payload struct {
+			Definition PropertyDefinition `graphql:"definition"`
+			Errors     []OpsLevelErrors   `graphql:"errors"`
+		} `graphql:"propertyDefinitionCreate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v, WithName("PropertyDefinitionCreate"))
+	return &m.Payload.Definition, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) GetPropertyDefinition(input IdentifierInput) (*PropertyDefinition, error) {
```

Comment:
```suggestion
func (client *Client) GetPropertyDefinition(input string) (*PropertyDefinition, error) {
```
Our interface should be flexible and take in a string and we try to push it to `IdentifierInput` using the construct that detects if its an ID or Alias